hello/App.tsx

```diff
@@ -46,33 +26,47 @@ const Section: React.FC<{
         style={[
           styles.sectionDescription,
           {
-            color: isDarkMode ? Colors.light : Colors.dark,
           },
         ]}>
         {children}
       </Text>
     </View>
   );
 };
+  
+const onLogin = async()=>{
+
+};
 
 const App = () => {
-  const isDarkMode = useColorScheme() === 'dark';
+  const [email, onChangeEmail] = React.useState("");
+  const [password, onChangePassword] = React.useState("");
 
-  const backgroundStyle = {
-    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
-  };
 
   return (
-    <SafeAreaView style={backgroundStyle}>
-        <View
-          style={{
-            backgroundColor: isDarkMode ? Colors.black : Colors.white,
-          }}>
-          <Section title="Primeiro programa">
-            Hello Wolrd!
+    <SafeAreaView>
+        <View>
+          <Section title="Bem-vindo(a) à Taqtile!">
           </Section>
+          <Text>
+            E-mail
+          </Text>
+          <TextInput
+            style={styles.input}
+            onChangeText = {onChangeEmail}
+            value = {email}
+          />
+          <Text>
+            Senha
+          </Text>
+          <TextInput
+            style={styles.input}
+            onChangeText = {onChangePassword}
+            value = {password}
+          />
+          <Button onPress={onLogin} title= "Entrar">
+          </Button>
```

Comment:
quando vc não tem nenhuma informação passando para dentro do componente vc pode fechar a tag direto na tag de abertura, por exemplo

```suggestion
          <Button onPress={onLogin} title= "Entrar" />
```